views/js/runner/plugins/security/fullScreen.js

```diff
@@ -251,31 +259,51 @@ define([
             }
 
             function alertUser() {
-                if (!waitingForUser) {
-                    if (fullScreenSupported) {
-                        waitingForUser = true;
-                        stopFullScreenChangeObserver();
-                        disableItem();
-                        testRunner.trigger('alert.fullscreen', message, function(reason) {
+                if (!waitingForUser && fullScreenSupported) {
+                    waitingForUser = true;
+                    stopFullScreenChangeObserver();
+                    disableItem();
+
+                    if(isIE11()) {
+                        testRunner.trigger('alert.fullscreen', message, userAlertCallbackIE11, dialogParams);
+                    } else if (fullScreenSupported) {
+                        testRunner.trigger('alert.fullscreen', message, userAlerCallback, dialogParams);
+                    }
```

Comment:
**thought:** I don't think the condition for `fullScreenSupported` is required. The check has been already made at the beginning, [line 262](https://github.com/oat-sa/extension-tao-test-runner-plugins/pull/183/files#diff-d72fcc91203b4bc43dcebe1640a9a3129c26b23d0782ccb8327469631962f63dR262)

**thought:** There is barely no difference between the 2 block codes, except for one parameter. I'd suggest using a variable instead

**thought:** the message should differ for the IE11 case.

**suggestion:**
```suggestion
                    const callback = isIE11() ? userAlertCallbackIE11 : userAlerCallback;
                    testRunner.trigger('alert.fullscreen', message, callback, dialogParams);
```